install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -407,6 +407,24 @@ bpf:
   # tracking table.
   # @default -- `262144`
   ctAnyMax: ~
+  # -- Control events generated by the Cilium datapath exposed to Cilium monitor and Hubble.
```

Comment:
The doc update is clear about

> Helm configuration for BPF events map rate limiting is experimental and might
    change in upcoming releases.

Can we also state here that the follow values are experimental please?

Reply:
Sure.